docs/examples/extension.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>do not need the <code>dict()</code>  call, the <code>{}</code>  creates the dict.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/halomod/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>